api/common/interfaces.go

```diff
@@ -129,6 +130,14 @@ func getLatestBlockHeight(s Serverer, params map[string]interface{}) map[string]
 	return respPacking(ledger.DefaultLedger.Blockchain.BlockHeight, SUCCESS)
 }
 
+func getNodeRelayMessageCount(s Serverer, params map[string]interface{}) map[string]interface{} {
```

Comment:
There is no need to add a new API endpoint for it. It would be good if people can get this result using `./nknc info -s`. To do that you just need to put it in LocalNode MarshalJSON method, together with height and version.